content/docs/09.5-security/15-self-managed-operation.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Self-Managed Operation"
```

Comment:
Per my discussion with @ritawatson we should see if we can get more information for this section OR drop it for the time being until it can be fully fleshed out. I think getting the SAAS portion finished would provide more benefit since on-site is always managed through an internal group and not something anyone can pickup and run.

Reply:
@JPedro2 @zulfilee Do you agree? Unless we have more information now for this section, such as ports that Palette needs access to and port traffic users would see, this section doesn't seem very informative to users. 

@dramich Alternatively, we can keep the section and add a line to say "On-prem deployments are managed through an internal group at Spectro Cloud. To set up Palette on-prem in your enterprise, contact ..."  
If we add such a blurb, what would the contact link be?